src/js/store/appState/actions.ts

```diff
@@ -6,7 +6,9 @@ import {
   SET_OPEN_LAYER_GROUP,
   SET_HIDE_WIDGET,
   AppState,
-  LeftPanel
+  LeftPanel,
+  SET_MEASURE_RESULTS,
```

Comment:
@vaidotasp @SwampGuzzler just flagging that @SwampGuzzler and I have chatted about moving measure results into the mapview reducer. Could this be a next PR?

Reply:
@eyosef This is out of scope for this PR; we can make an Issue afterwards!

Reply:
new issue!